### PR TITLE
.

### PR DESCRIPTION
**Pull request by ahmedalhamad7**
Please approve my pull request

### CODE REVIEW

File: include.js [hunk 1]
```diff
@@ -6,6 +6,15 @@ let cards = [
...
   },
+  let cards = [
+  {
+    artName: 'Moonrise', 
+    pageLink: './Art/ahmedalhamad7/moon.html', 
+    imageLink: './Art/ahmedalhamad7/static moon.png', 
+    author: 'Ahmed', // use your name
+    githubLink: 'https://github.com/ahmedalhamad7'
+  }
+];
```
Comment (sophiabrandt) [inline include.js:17]: This will break the code.
Please remove line 9 & line 17.

Please fix this.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
Thanks you for this contribution. 🎉 

Please check the following checklist that you need to fulfill for us to be able to merge your request.

## Issues: 

- [ ]  fix the syntax errors in your `include.js` entry 
- [ ] (optional, but recommended): crop the gif/png to your animation, remove the white space

If you have questions feel free to reach to the discord or ask them here.

File: include.js (hunk 1 above)
Comment (ahmedalhamad7) [reply include.js]: I did the changes as requested, sorry for the mistake, and thanks for notifying.

**Comment by sophiabrandt** [comment]
@ahmedalhamad7 please let me know if you're finished with your changes.

For example, you didn't complete the optional task. While I can merge this now, I don't know if you still want to do that.

**Comment by ahmedalhamad7** [comment]
i have re uploaded the photo to get rid of white spaces

**Pull request merged**